pkgs/top-level/all-packages.nix

```diff
@@ -20811,7 +20811,7 @@ with pkgs;
 
   nix-pin = callPackage ../tools/package-management/nix-pin { };
 
-  nix-prefetch-github = callPackage ../build-support/nix-prefetch-github {};
+  nix-prefetch-github = with python3.pkgs; toPythonApplication nix-prefetch-github;
```

Comment:
By the way we also have [nix-prefetch](https://github.com/msteen/nix-prefetch), which can handle a lot more.